modules/edu.gemini.seqexec.web/edu.gemini.seqexec.web.client/src/main/scala/edu/gemini/seqexec/web/client/components/sequence/steps/StepsTableContainer.scala

```diff
@@ -277,7 +276,7 @@ object StepsTableContainer {
       }
 
     private def classSet(step: Step): List[(String, Boolean)] = List(
-      "disabled" -> (step.skip || step.status === StepState.Completed),
+      "disabled" -> (step.status === StepState.Completed),
```

Comment:
I'm not sure about this change. It gives the same background to the skipped states than the regular steps. Perhaps `disabled` isn't appropriate as we use it for completed steps but we should mark it somehow differently

Reply:
Skipped steps and successfully completed steps are similar, in that both kinds are already run and have are in a final state.